bindata/network/ovn-kubernetes/common/003-rbac-controller.yaml

```diff
@@ -106,6 +106,20 @@ rules:
   - patch
   - update
   - watch
+- apiGroups: ["policy.networking.k8s.io"]
+  resources:
+  - adminnetworkpolicies
+  - baselineadminnetworkpolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["policy.networking.k8s.io"]
+  resources:
+  - adminnetworkpolicies/status
+  - baselineadminnetworkpolicies/status
+  verbs:
+  - update
```

Comment:
Is cluster-manager updating `adminnetworkpolicies/status`, `baselineadminnetworkpolicies/status` together with ovnkube-node?

Reply:
so this stemmed from confusion - this file should be called `003-rbac-ovnkube-control-plane`? 
s/ovnkube-controller/ovnkube-node/ovnkube-control-plane
changing this to removing perms from this file. control plane doesn't do anything to anp/banp today